### PR TITLE
add Documentation and Contact Us link, switch up toolbar links in staging

### PR DESCRIPTION
**Pull request by wslack**
After this is merged, move https://github.com/18F/federalist/issues/536 to Done?

### CODE REVIEW

File: assets/app/components/nav.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ const propTypes = {
 const getSecondaryNav = (loggedIn, username) => {
   const authLinks = [
     <Link to="/sites">{username}</Link>,
+    <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
+    <a href="https://github.com/18F/federalist/issues/new" target="_blank">Contact us</a>
```
Comment (jmhooper) [inline assets/app/components/nav.js]: Looks like there's a syntax error from a missing comma at the end of this line.

Also, note that ES6 gives us [trailing commas](http://dontkry.com/posts/code/trailing-commas.html), so if you wanted you could also stick a comma to the end of line 17 below to reduce diff noise and make issues with commas more obvious in the future.

File: assets/app/components/nav.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ const propTypes = {
 const getSecondaryNav = (loggedIn, username) => {
   const authLinks = [
     <Link to="/sites">{username}</Link>,
+    <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
```
Comment (jmhooper) [inline assets/app/components/nav.js]: In the unauthenticated view, this link is titled "Help", but here it is titled "Documentation". Not asking for a change here, just wanted to make sure we realize and are okay with that.

File: assets/app/components/nav.js (hunk 1 above)
Comment (wslack) [inline assets/app/components/nav.js]: edits done!

File: assets/app/components/nav.js (hunk 2 above)
Comment (wslack) [inline assets/app/components/nav.js]: changed the other "help" to "documentation"

**Comment by jmhooper** [comment]
I built this and the root URL was still showing "Help" instead of "Documentation". Looks like it may also need to be updated [here](https://github.com/18F/federalist/blob/staging/assets/index.html#L58).

**Comment by wslack** [comment]
Should we delete some of the code on the other page if its not being used?

**Comment by jmhooper** [comment]
We probably can, but I'm not sure that it is not being used. Let's remove it and see what happens.

**Comment by wslack** [comment]
you or me?

**Comment by jmhooper** [comment]
Alright, so it looks like `unAuthLinks` is only used if the `Nav` component's `isLoggedIn` prop is false. That prop is passed through the `Header` component:

```javascript
const Header = ({ isLoggedIn = false, username = null }) =>
  <header className="usa-site-header" role="banner">
    <Disclaimer />
    <Nav isLoggedIn={ isLoggedIn } username={ username } />
  </header>
```

Which has it's props passed down from the `App` component where the logic to set `isLoggedIn` lives:

```javascript
render() {
  const storeState = this.state;

  return (
    <div>
      <Header
        isLoggedIn={ !!storeState.user }
        username={storeState.user.username}
      />
      <!-- ... -->
    </div>
  );
}
```

What's interesting about this, it that `main.js`, the entry point for the frontend has a check for the presence of `storeState.user` and does not render if that is false:

```javascript
store.subscribe(() => {
  const state = store.getState();

  if (!state.user) return;

  render(/* ... */);
});
```

What this means is this code is in fact never used since `isLoggedIn` is only false if there is no user in the state, and if there is no user in the state the frontend will not render.

**Comment by wslack** [comment]
Neat. Syntax wise, is it better just to comment it out with a note to refer to the HTML

**Comment by jmhooper** [comment]
I think we should remove it completely if it is dead code. I started the work and it looks like removing the logic for  `isLoggedIn` breaks some tests. With that in mind I advise merging this and moving that work to another branch.

**Comment by jmhooper** [comment]
The PR to remove the `isLoggedIn` logic is open at #704

**Pull request merged**